pkg/registry/apis/provisioning/import.go

```diff
@@ -80,48 +79,12 @@ func (c *importConnector) Connect(
 		ref := r.URL.Query().Get("ref")
 		logger := logger.With("ref", ref)
 
-		tree, err := repo.ReadTree(r.Context(), logger, ref)
-		if err != nil {
-			logger.ErrorContext(ctx, "failed to read tree", "error", err)
-			responder.Error(apierrors.NewInternalError(fmt.Errorf("failed to read tree: %w", err)))
+		if err := replicator.ReplicateTree(r.Context(), ref); err != nil {
```

Comment:
do we still need "import"?

IIUC, it can/should queue an async sync job (just like the /webhook does)

(not required for htis PR!)

Reply:
yeah, that's what the next PR is about :)